client/state/followers/test/selectors.js

```diff
@@ -1,12 +1,11 @@
-/**
- * External dependencies
- */
-import { expect } from 'chai';
-
 /**
  * Internal dependencies
  */
-import { getFollowersForQuery } from '../selectors';
+import {
+	getFollowersForQuery,
+	isFetchingFollowersForQuery,
+	getTotalFollowersForQuery,
+} from '../selectors';
 import { getSerializedQuery } from '../utils';
 
 describe( 'selctors', () => {
```

Comment:
There's a pre-existing typo here:

```suggestion
describe( 'selectors', () => {
```

If you ask me, this `describe` is unnecessary, it's clear that this is testing selectors when seeing the name of this file.